.github/workflows/test-suite.yml

```diff
@@ -309,7 +309,6 @@ jobs:
       run: |
           make
     - name: Install lcli
-      if: env.SELF_HOSTED_RUNNERS == 'false'
```

Comment:
I had to remove this line since I added a new flag to `lcli` and it wasn't being picked up.